jpw1991/chebs-necromancy
# 2.1.1: Add bonus minions for necromancy level if configured to use limits

Compiled: [here](https://github.com/jpw1991/chebs-necromancy/releases/tag/2.1.1)

**Testing:**

Please tick the box as you complete each test segment.

- [x] **Base functionality test**
  - Delete your config file
  - Start game
  - Ensure that you can summon unlimited minions
  - Quit game
- [x] **Limit increment test**
  - Edit config file to impose a limit on skeletons and draugr (different limit for each)
  - Edit the incrementer for each minion type (or leave it as default of 10)
  - Start game and check that you are being permitted to get new minions each time you get another X levels of Necromancy skill
  - Quit game
- [x] **Limit no increment test**
  - Edit config file to have incrementers set to 0
  - Ensure that the limit can't be passed and that no extra minions are given as necromancy level increases
- [x] **Mixture test**
  - To be exceedingly thorough, test incrementer with skeletons works well without an incrementer on the draugr, and vice versa
